linkedevents/settings.py

```diff
@@ -436,9 +437,74 @@ def sentry_anonymize_user_repr(obj, hint):
         "events.auth.ApiTokenAuthentication",
     ),
     "DEFAULT_VERSIONING_CLASS": "rest_framework.versioning.URLPathVersioning",
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
     "VIEW_NAME_FUNCTION": "linkedevents.utils.get_view_name",
 }
 
+SPECTACULAR_SETTINGS = {
+    "TITLE": "Linked Events information API",
+    "DESCRIPTION": (
+        "Linked Events provides categorized data on events and places using JSON-LD format."
+        "\n\n"
+        "Events can be searched by date and location. Location can be exact address or larger"
+        "area such as neighbourhood or borough."
+        "\n\n"
+        "JSON-LD format is streamlined using include mechanism. API users can request that certain"
+        "fields are included directly into the result, instead of being hyperlinks to objects."
+        "\n\n"
+        "Several fields are multilingual. These are implemented as object with each language variant"
+        "as property. In this specification each multilingual field has (fi,sv,en) property triplet as"
+        "example."
+    ),
+    "VERSION": None,
+    "SCHEMA_PATH_PREFIX": r"/v1/",
+    "SCHEMA_PATH_PREFIX_TRIM": True,
+    "PREPROCESSING_HOOKS": ["linkedevents.schema_utils.swagger_endpoint_filter"],
+    "POSTPROCESSING_HOOKS": ["linkedevents.schema_utils.swagger_postprocessing_hook"],
+    "GET_LIB_DOC_EXCLUDES": "linkedevents.schema_utils.swagger_get_lib_doc_excludes",
+    "SERVERS": [
+        {"url": "https://api.hel.fi/linkedevents/v1"},
```

Comment:
actually also I pondered those "SERVERS" in the first place that should we include them (because of maintenance) and came to conclusion it is a nice to have 👍

Reply:
Added the missing slash as it should have had that.